ebos/eclproblem.hh

```diff
@@ -2874,22 +2863,22 @@ private:
                     std::vector<bool>* data = nullptr;
                     switch (bcface.dir) {
                     case FaceDir::XMinus:
-                        data = &freebcXMinus_;
+                        data = &freebc_.minus[0];
                         break;
                     case FaceDir::XPlus:
-                        data = &freebcX_;
+                        data = &freebc_.plus[0];
                         break;
                     case FaceDir::YMinus:
-                        data = &freebcYMinus_;
+                        data = &freebc_.minus[1];
                         break;
                     case FaceDir::YPlus:
-                        data = &freebcY_;
+                        data = &freebc_.plus[1];
                         break;
                     case FaceDir::ZMinus:
-                        data = &freebcZMinus_;
+                        data = &freebc_.minus[2];
                         break;
                     case FaceDir::ZPlus:
-                        data = &freebcZ_;
+                        data = &freebc_.plus[2];
                         break;
```

Comment:
Why not get rid of this switch-case by using a container for both plus and minus that allows lookup with `bcface.dir`like `data = freebc.data[bcface_dir]`. But maybe that is not worth it as there are other swich-cases that won't be affected.

Reply:
i did consider it, but found it too much for too little gain. but i can do it if you insist.

Reply:
Nope. I have the same impression now.